systemtest/src/main/java/io/strimzi/systemtest/security/SystemtestCertManager.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.security;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+import static java.util.Arrays.asList;
+
+public class SystemtestCertManager {
```

Comment:
I will go for this one...
```suggestion
public class SystemTestCertManager {
```